com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/filter/ConvertedQuery.java

```diff
@@ -87,7 +87,7 @@ public void addParameter(MColumn column, String parameter) {
 					!column.getColumnName().equals("EntityType") && 
 					!column.getColumnName().equals("AD_Language"))
 				addParameter(Integer.valueOf(parameter));
-			else if (displayType == DisplayType.Integer)   //   Integer
+			else if (displayType == DisplayType.Integer || displayType == DisplayType.Button)   //   Integer
```

Comment:
OK, there is a mistake here and in NumericTypeConverter:71

Most of buttons are strings, from what I see for a button to be numeric the columnname must end with _ID, so is better to change the code to verify this condition, otherwise this code can potentially fail for string buttons.

Reply:
@igorpojzl pls take a look

Reply:
Added control for ColumnName ending with "_ID"

Reply:
Thanks @igorpojzl - I think the same change is still required in NumericTypeConverter:71

Reply:
@CarlosRuiz-globalqss Hi i have one question, should i fix it in NumericTypeConverter:71, or TypeConverterUtils:182, or both.